### PR TITLE
add shortname for authz

### PR DESCRIPTION
**Pull request by zirain**
I'm not sure is this raised before, but this will save a little life for me, `authorizationpolicies` is too long.

### CODE REVIEW

File: security/v1/authorization_policy.proto [hunk 1]
```diff
@@ -449,7 +449,7 @@ option go_package="istio.io/api/security/v1";
 // +cue-gen:AuthorizationPolicy:labels:app=istio-pilot,chart=istio,istio=security,heritage=Tiller,release=istio
 // +cue-gen:AuthorizationPolicy:subresource:status
 // +cue-gen:AuthorizationPolicy:scope:Namespaced
-// +cue-gen:AuthorizationPolicy:resource:categories=istio-io,security-istio-io,plural=authorizationpolicies
+// +cue-gen:AuthorizationPolicy:resource:categories=istio-io,security-istio-io,shortNames=authz,plural=authorizationpolicies
```
Comment (hanxiaop) [inline security/v1/authorization_policy.proto:452]: I think this can be `ap` or follow some similar patterns like other security resources.

**Comment by hzxuzhonghu** [comment]
Yes , proposed before. https://github.com/istio/api/pull/1378

I am ok with this

**Review by hzxuzhonghu — APPROVED** [review]

**Review by linsun — COMMENTED** [review]
probably need more TOC member to review. I think we also need a small release note for this.

File: security/v1/authorization_policy.proto (hunk 1 above)
Comment (linsun) [reply security/v1/authorization_policy.proto:452]: agree `ap` may be better

Comment (zirain) [reply security/v1/authorization_policy.proto]: updated

**Review by linsun — COMMENTED** [review]
LGTM, wait for other TOC members to review

**Review by nrjpoddar — APPROVED** [review]
LGTM

**Pull request merged**